#Commented out the line about libcap in order to make vsftpd compilable.

diff -Naur vsftpd-2.0.3/sysdeputil.c vsftpd-2.0.3.new/sysdeputil.c
--- vsftpd-2.0.3/sysdeputil.c	2008-01-26 13:20:09.000000000 +0100
+++ vsftpd-2.0.3.new/sysdeputil.c	2008-01-26 13:15:50.000000000 +0100
@@ -158,7 +158,7 @@
 #include <linux/capability.h>
 #include <errno.h>
 #include <syscall.h>
-_syscall2(int, capset, cap_user_header_t, header, const cap_user_data_t, data)
+//_syscall2(int, capset, cap_user_header_t, header, const cap_user_data_t, data)
 /* Gross HACK to avoid warnings - linux headers overlap glibc headers */
 #undef __NFDBITS
 #undef __FDMASK
